proto/messages.proto

```diff
@@ -218,5 +218,5 @@ enum ClientAction {
 }
 
 message Calling {
-  required string content = 1;
+  required bytes content = 1;
```

Comment:
AFAIK there is no client out there, not even internal, other than the AVS command line test app, that sends this messages, so it should be safe to change it. Android ( @deanrobertcook ) and Web ( @gregor ) are still far from implementing this, right? And we are not using this on iOS. If AVS agrees to change it on their command line tool, then no need to keep this around. I would prefer not to accumulate deprecations when not needed.